### ISSUE TITLE
slow to open

### ISSUE THREAD

**Issue by rie39414** [standardnotes/forum#3589]
version 3.194.5 with appimage on ubuntu 24.04

the application opens very slowly, it takes about 7 seconds, even with the flatpack version and the windows 10 and 11 version

it takes longer on my machines than gimp or photoshop and I should open the webapp on firefox first to take notes but I would need a better opening time so I can also use it offline as desktop app

**Comment by effieeee** [comment]
Hi @rie39414, while I haven't been able to test this on Linux, I'm afraid I haven't encountered that long of a loading time on the same version of the app for Windows 11 so far. To clarify though, were you referring to the loading time for **launching** the app, or were you referring to the time it takes to **decrypt and load your notes** on the app, after it has launched?

If it's the latter, please note that databases that reach up to the 1000s can start to experience significantly longer loading times when decrypting the notes and items on the app.

**Comment by rie39414** [comment]
no, I'm referring precisely to the app startup time from the moment it is closed, I have a desktop (windows 10 and ubuntu 24) with i5-6600 and a laptop (windows 11 and ubuntu 24) with ryzen 3500u

**Comment by effieeee** [comment]
I see, thank you for clarifying. Just to rule out any issues with your present installation, try a clean reinstall of the desktop app on Windows first 🙏 You may do this by following the [uninstallation instructions](https://standardnotes.com/help/72/how-do-i-uninstall-standard-notes) on our help page, then download and install the latest version of the app for your device on our website’s [homepage](https://standardnotes.com/).
